src/interfaces/assistants_web/tailwind.config.js

```diff
@@ -116,8 +116,6 @@ module.exports = {
         },
         DEFAULT: {
           css: {
-            'code::before': { content: '""' },
```

Comment:
why remove this? This is old school stuff for `float`.

Reply:
this removes the quotes that get added to `blockquotes` in markdown. We can keep them, just thought it would be cleaner if we displayed exactly what the model outputs

Reply:
oh wait, this I added on my previous PR. It was merged unintentionally so I am just reverting it

Reply:
👍 i dunno what im talking about. haha.